tensorflow/examples/tutorials/mnist/fully_connected_feed.py

```diff
@@ -251,9 +251,9 @@ def main(_):
       help='Batch size.  Must divide evenly into the dataset sizes.'
   )
   parser.add_argument(
-      '--train_dir',
+      '--data_dir',
```

Comment:
something like this?
https://github.com/tensorflow/tensorflow/blob/master/tensorflow/examples/tutorials/mnist/mnist_with_summaries.py#L201